index.js

```diff
@@ -1,7 +1,8 @@
 import { default as Usbmux } from './lib/usbmux';
 import * as utilities from './lib/utilities';
 import * as services from './lib/services';
+import { SERVICE as INSTRUMENT_SERVICE } from './lib/instrument';
```

Comment:
I'm not quite sure where this is exported. Shouldn't all services be started using start...Service helpers?

Reply:
This usage is over `services.startInstrumentService`.

e.g. https://github.com/appium/appium-xcuitest-driver/pull/1403/files#diff-cadf38942bfa2205c3e85f7e37de81f7dc552069916c98d45b2be3b06ab4d544R97

```
    const instrumentService = await services.startInstrumentService(this.opts.device.udid);
    const processes = await instrumentService.callChannel(SERVICE.DEVICE_INFO, 'runningProcesses');
```

So, this is not service itself, but used over service.
Hm... I wonder `CHANNEL`, `INSTRUMENT_CHANNEL` is better naming than `SERVICE` to reduce confusion.

currently the xcuitest driver refer to this value via `import { SERVICE } from 'appium-ios-device/build/lib/instrument';`, but probably `import { services, INSTRUMENT_SERVICE } from 'appium-ios-device';` in xcuitest more make sense...

Reply:
renamed.

Reply:
I like CHANNEL better over INSTRUMENT_CHANNEL, but both would work